src/libcmd/repl.cc

```diff
@@ -422,8 +422,6 @@ StringSet NixRepl::completePrefix(const std::string & prefix)
             // Quietly ignore parse errors.
         } catch (EvalError & e) {
             // Quietly ignore evaluation errors.
-        } catch (UndefinedVarError & e) {
-            // Quietly ignore undefined variable errors.
```

Comment:
What's this?

Reply:
`UndefinedVarError` used to be a subclass of `Error`. This PR changes it to be a subclass of `EvalError` instead. There's already a `catch` clause for `EvalError` here, so the `UndefinedVarError` clause is no longer needed.

Reply:
OK sounds good.